internal/distro/rhel8/distro.go

```diff
@@ -847,7 +847,7 @@ func New() distro.Distro {
 		},
 		kernelOptions: "console=ttyS0 console=ttyS0,115200n8 no_timer_check crashkernel=auto net.ifnames=0",
 		bootable:      true,
-		defaultSize:   2 * GigaByte,
+		defaultSize:   4 * GigaByte,
```

Comment:
I think this is right if the default images really do need more than 4GB. But this does not only change the test cases, but the actual defaults we provide, so should update the documentation too (each image type has high-level docs). And the commit message should probably reflect that. I assume the actual test cases are regenerated in a separate commit?

Reply:
@teg The test case changes are in [`de1e3c6` (#882)](https://github.com/osbuild/osbuild-composer/pull/882/commits/de1e3c6d18e7d3ce853b640cdc0b71a516efb122).

I'll update the docs/commit in a short while.

Reply:
For what it's worth, we don't mention image sizes in the docs right now (but that might be a good thing for the future.

Reply:
My bad! Could you just update the commit message, so it doesn't look like this is just about changing the tests?

```
distro/rhel8: increase default size to 4GB

RHEL 8.3 requires more than 2GB to install on aarch64/ppc64le.
```
or something to that effect?

Reply:
Sure!

Reply:
@teg Done!